BUILD.bazel

```diff
@@ -356,6 +356,27 @@ genrule(
     executable = 1,
 )
 
+genrule(
```

Comment:
Taking from `go.mod`:

```
github.com/onsi/ginkgo v1.12.1
```

which is eactly where we have our tooling library from. If we have issues with it or want to update it, it can be done in `go.mod` like before.